### ISSUE TITLE
Re-add popup.html

### ISSUE THREAD

**Issue by NajmAjmal** [alexlostorto/magic-notes#6]
Hello, I had a suggestion: could you please re-add the popup.html? and include a download button that downloads the data (bookwork codes with answers are stored with them)

If not, could you aid me in adding this feature on my version of "Sparxmaths Solver". You can submit a Pull request on the [CDN](https://github.com/SintcoLTD/CDN/blob/main/chrome-extension/sparxmaths/console/console-2.js) or on the [Main extension page](https://github.com/NajmAjmal/sparxmaths)

Thanks, Najm

**Comment by alexlostorto** [comment]
I see the code for the Download button already in your repo (popup.js).

<img width="399" alt="image" src="https://user-images.githubusercontent.com/114829362/236049143-3e76ee22-796f-4440-8865-5431e801a308.png">

This feature is already on your version of the program... Are you suggesting it as an addition to my program?

**Comment by NajmAjmal** [comment]
Yes, I see the code in my program, but it does not function. If you can aid me in making this work, i would appreciate it. When the download button is clicked in "popup.html" it so far does nothing. Maybe it can not reach the console.js or MongoDB.

I am not suggesting this as a addition to your program.

Thanks, Najm

**Comment by alexlostorto** [comment]
Hmmmm, the download button is not connected to MongoDB. 

I would try a few things:
- Check that you are using Chrome
- Check that the localStorage key 'sparxCodes' exists and contains data (do this by console.logging the database)
- Look at the console or background service worker to see if there are any errors (make sure to temporarily remove the recurring credits function)
- Make sure that the event listener is actually connected to the download button and is triggered.
